src/wpt/WebCryptoAPI-test.ts

```diff
@@ -213,7 +213,7 @@ export default {
   },
   'idlharness.https.any.js': {
     comment: 'Test file /resources/WebIDLParser.js not found.',
-    skipAllTests: true,
+    skippedTests: 'ALL',
```

Comment:
What is the difference between skipped and excluded tests? It's really ambiguous as a non-native english speaker.